app/services/authorizer.rb

```diff
@@ -104,8 +104,8 @@ def build_filtered_scope_components(resource_class, all_filters, options)
   def build_scoped_search_condition(filters)
     raise ArgumentError if filters.blank?
 
-    strings = filters.map { |f| "(#{f.search_condition.presence || '1=1'})" }
-    strings.join(' OR ')
+    filters.reject! { |f| f.search_condition.blank? }
+    filters.map { |f| "(#{f.search_condition})" }.join(' OR ')
```

Comment:
Wouldn't it be better to avoid modification in place? Might as well chain it altogether, right?